build.gradle

```diff
@@ -23,6 +23,10 @@ intellij {
 dependencies {
     compile group: 'org.eclipse.lsp4j', name: 'org.eclipse.lsp4j', version: '0.6.0'
     compile group: 'com.vladsch.flexmark', name: 'flexmark', version: '0.34.58'
+    testCompile group: 'junit', name: 'junit', version: '4.11'
+    testCompile group: 'org.mockito', name: 'mockito-all', version: '1.6.5'
+    testCompile group: 'org.powermock', name: 'powermock-api-mockito', version: '1.6.5'
+    testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: '1.6.5'
```

Comment:
I think the versions are clashing.. I am not sure how it works.. This is the wiki for mocking https://github.com/powermock/powermock/wiki/mockito

Mainly stating that powermock has to be a lower version than mockito

Reply:
If you are adding both maybe only use Powermock?